ansible_rulebook/schema/ruleset_schema.json

```diff
@@ -75,6 +75,25 @@
                         {"$ref": "#/$defs/none-action"},
                         {"$ref": "#/$defs/shutdown-action"}
                     ]
+                },
+                "once_within" : {
+                    "type": "string",
+                    "minLength": 6,
+                    "pattern": "^\\d+\\s{1}(milliseconds|seconds|minutes|hours|days)$"
```

Comment:
I guess that singular values are not supported, right? for example: `1 day`, `1 hour` etc...
This should be emphasized in the documentation

Reply:
@Alex-Izquierdo it supports both plural and singular form. It doesn't check if singular digit values are truly unit values. Like 1 day versus 1 days.

Reply:
Ok! thanks for the clarification